trunk/js/popup.js

```diff
@@ -137,6 +137,17 @@
 				}
 
 				window.send_to_editor( formval );
+				
+				$.ajax({
+					type     : 'post',
+					dataType : 'html',
+					url      : PardotShortcodePopup.ajaxurl,
+					data     : {
+						action : 'delete_asset_html_transient',
+						asset_id : formval.match(/id="([0-9]*)"/)[1],
+						asset_type : 'form'
+					}
+				});
```

Comment:
This is where we call the newly create AJAX function that clears out the cached HTML for the form when we select it in the classic editor.